test/testdata/lsp/rubyfmt_init_error.rb

```diff
@@ -0,0 +1,2 @@
+# typed: strict
+# rubyfmt-force-exit 101
```

Comment:
If it's not too hard, I'd love this to follow the same style as other assertions and have a colon

```ruby
# rubyfmt-force-exit: 101
```